src/Store.js

```diff
@@ -33,7 +35,8 @@ class Store {
 
     // Create IDs, names and paths
     this.id = address.toString()
-    this.uid = peerId
+    this.identity = identity
+    this.uid = identity.publicKey
```

Comment:
I wonder if we can remove `uid` completely. I don't think we use it anywhere anymore.